content/en/getting_started/synthetics/api_test.md

```diff
@@ -0,0 +1,62 @@
+## Create an API test
+
+[API tests][1] help you monitor your API endpoints and alert you when they are failing or too slow. These checks verify that your applications are responding to requests and meet any conditions you define, such as response time, HTTP status code, and header or body contents. Use the [Datadog API][2] to see the full list.
+
+In this example, an API test is created to ensure your website is constantly up and providing responses in a specific amount of time.
+
+### Configure the request
+
+1. In the Datadog application, hover over **[UX Monitoring][3]** in the left hand menu and select **Synthetics Test**. 
+2. In the top right corner, click the **New Test** button. 
+3. Select **API test**.
+4. Define the configuration of your API test:
+    - Add the URL of the endpoint you want to monitor. If you don’t know what to start with, you can use https://www.shopist.io/ which is a test web application.”
+    - Select **Advance Options** to use custom request headers, authentication credentials, body content, or cookies.
+    - Add one or two tags, like `prod`, to help organize and filter tests.
```

Comment:
should we say "some" tags? this wording makes me think you must add either one or two tags in order for this to work